client/me/purchases/purchases-site/index.jsx

```diff
@@ -32,6 +32,8 @@ const PurchasesSite = ( {
 } ) => {
 	let items;
 
+	const isJetpack = ! isPlaceholder && some( purchases, purchase => isJetpackPlan( purchase ) );
+
```

Comment:
Any idea why this hasn't been using `isJetpackSite( state, siteId )` in `connect()`? Would it make sense to change it now?

Reply:
We may not have a site object at all! `isJetpackSite` depends on having a site loaded. Checking the purchase is the only reliable was to do it here where a site may not exist.